literature/wang_1996_ordered_6672/Wang_1996_Ordered_6672.bib

```diff
@@ -0,0 +1,10 @@
+@article{wang_1996_ordered_6672,
```

Comment:
Actually, this file should be removed from the PR. It has capital letters and the second version in lower case was also submitted below.

Reply:
Sure, I delete the duplicate.